src/scripts/evaluate_llm_benchmark.py

```diff
@@ -7,13 +7,28 @@
 import numpy as np
 import pandas as pd
 from omegaconf import DictConfig
+from sklearn.preprocessing import FunctionTransformer
 
 from european_values.data_loading import load_evs_trend_data, load_evs_wvs_data
 from european_values.data_processing import process_data
 
 logger = logging.getLogger("evaluate_llm")
 
 
+def sigmoid_transform(log_likelihoods, alpha=0.05, center=-50.0):
```

Comment:
I think this should be defined during training of the model, and ideally the `alpha` and `center` parameters should be fitted during training as well. It should then be stored in a pipeline to disk, just like the model is. In this `evaluate_llm_benchmark` script we can then just load in the fitted model and transform, and use them.